.circleci/config.yml

```diff
@@ -221,7 +221,6 @@ jobs:
       - when:
           condition: << parameters.spm >>
           steps:
-            - run: for ip in $(dig @8.8.8.8 github.com +short); do ssh-keyscan github.com,$ip; ssh-keyscan $ip; done 2>/dev/null >> ~/.ssh/known_hosts
```

Comment:
This removes the workaround in https://github.com/mapbox/mapbox-navigation-ios/pull/2771#discussion_r567167979.

Reply:
CI job seems still [failing](https://circleci.com/gh/mapbox/mapbox-navigation-ios/26105?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link) on that ssh fingerprint issue. Are there any other ssh'ed dependencies? Maybe it's too early to remove this workaround?

Reply:
I don’t think there are any other SSH’d dependencies. I wonder if the CI machine is caching something.

Reply:
I’ve restored the workaround. It should be harmless anyways.